rules/legacy/tests/Rector/Include_/RemoveIncludeRector/Fixture/fixture.php.inc

```diff
@@ -21,13 +20,10 @@ namespace Rector\Legacy\Tests\Rector\Include_\RemoveIncludeRector\Fixture;
 
 class SomeClass
 {
-
     private function someMethod() : void
     {
-        // comment before include
-
         // comment after include
```

Comment:
For sake of consistency with other node removal, the comment above is attatched to it, so it should be removed too